app/views/schools/meters/index.html.erb

```diff
@@ -27,9 +36,14 @@
           <td><%= meter.mpan_mprn %></td>
           <td><%= meter.name %></td>
           <td><%= meter.meter_type.humanize %></td>
+          <td><%= meter.amr_data_feed_readings.count %></td>
+          <td><%= meter.amr_validated_readings.count %></td>
           <td>
             <div class="btn-group">
               <%= link_to 'Edit', edit_school_meter_path(@school, meter), class: 'btn btn-sm btn-warning' %>
+              <% if current_user.admin? && meter.amr_validated_readings.any? %>
```

Comment:
This checks the user is an admin because that's what the reports controller checks (i.e. it doesn't use cancan). I can look changing that and updating this check as part of the permissions review task